test-data/unit/check-kwargs.test

```diff
@@ -491,3 +491,11 @@ m = {} # type: Mapping[str, object]
 f(**m)
 g(**m)  # TODO: Should be an error
 [builtins fixtures/dict.pyi]
+
+[case testPassingEmptyDictWithStars]
+def f(): pass
+def g(x=1): pass
+
+f(**{})
+g(**{})
+[builtins fixtures/dict.pyi]
```

Comment:
Can you also add a test case where we check the this continues to generate an error:

```
def f(**kwargs: int) -> None: pass

f(**{'x': 'y'})  # Here should be an error
```